src/libraries/pretest.proj

```diff
@@ -63,10 +63,10 @@
           Outputs="$(MicrosoftNetCoreAppRuntimePackDir)data\PlatformManifest.txt"
           Condition="'$(BuildTargetFramework)' == '$(NetCoreAppCurrent)' or '$(BuildTargetFramework)' == ''">
     <GenerateFileVersionProps Files="@(SharedFrameworkRuntimeFile)"
-                              PackageId="$(MicrosoftNetCoreAppFrameworkName).Runtime.$(PackageRID)"
+                              PackageId="$(MicrosoftNetCoreAppFrameworkName).Runtime.$(OutputRid)"
```

Comment:
This changes the non-shipping "fake" runtime pack id. Do we also make changes to the actual shipping runtime pack sfxproj? I'm suprised that there aren't any changes to the sfxprojs.

Reply:
I wasn't aware there's a fake runtime pack id, and a shipping runtime pack id.
Do these represent different builds?

I assume the sfxprojs pick up the `MicrosoftNetCoreAppRuntimePack*` properties?